site/_posts/2020-06-03-user-registration-and-sign-in-with-laravel.md

```diff
@@ -1,11 +1,12 @@
 ---
 layout: blog-post
 title: User registration and sign-in with Laravel and FusionAuth
-description: We'll build a Laravel application that delegates user management, registration and login to FusionAuth.
+description: We'll build a Laravel application that delegates user management, registration, and login to FusionAuth.
 author: Karl Hughes
 image: blogs/fusionauth-laravel/user-registration-sign-in-laravel-fusionauth.png
 category: blog
-tags: client-php
+updated-date: 2020-30-03
+tags: client-php, tutorial, tutorial-laravel
```

Comment:
Please add `tutorial-php` as well. Probably should add `tutorial-javascript` to the vue blog post too. My bad for forgetting.

Reply:
Makes sense!